test/common/init/manager_impl_test.cc

```diff
@@ -49,18 +49,21 @@ TEST(InitManagerImplTest, AddReadyTarget) {
 
   ExpectableWatcherImpl w;
 
+  t1.expectInitialize();
```

Comment:
@mergeconflict add this not only because it is easier to coordinate but also less astonishment.

Reply:
Reason: 
`AddTargetAndMarkReadyBeforeInitialization` does call `expectInitialize`
Prior to this PR this function is also called anyway.